content/doc/book/pipeline/syntax.adoc

```diff
@@ -697,6 +697,13 @@ pipeline {
 
 The `when` directive allows the Pipeline to determine whether the stage should
 be executed depending on the given condition.
+The `when` directive must contain at least one condition.
+If the `when` directive contains more than one condition,
```

Comment:
It's unclear what this is about. How could a `when` contain more than one condition, if not using the nested conditions? What's the difference between this and `allOf`?